yt_dlp/extractor/drtv.py

```diff
@@ -236,9 +238,9 @@ def _real_extract(self, url):
                 'thumbnail': ('images', 'wallpaper'),
                 'release_timestamp': ('customFields', 'BroadcastTimeDK', {parse_iso8601}),
                 'duration': ('duration', {int_or_none}),
-                'series': ('season', 'show', 'title'),
+                'series': ('show', 'title'),
```

Comment:
This breaks shows such as https://www.dr.dk/drtv/se/frank-and-kastaniegaarden_71769
Currently:
```
> yt-dlp "https://www.dr.dk/drtv/se/frank-and-kastaniegaarden_71769" --print series
Frank & Kastaniegaarden
```
With your change
```
> yt-dlp "https://www.dr.dk/drtv/se/frank-and-kastaniegaarden_71769" --print series
NA
```

You can change the traversal to follow both paths with something like `((None, 'season'), 'show', 'title', {str}, any),`
Please also add an appropriate test.